fastify.js

```diff
@@ -266,7 +266,12 @@ function fastify (options) {
   })
   // Override to allow the plugin incapsulation
   avvio.override = override
-  avvio.on('start', () => (fastify[kState].started = true))
+  avvio.on('start', () => {
+    hookRunnerApplication('onCompilerReady', fastify, () => {
```

Comment:
I put this runner here because the `*Compiler`s are set in `route.js` in the `preReady` event.

So, I can't use:
- `preReady`: since it would be enqueued first of the function that does the job
- `after`: same before

Moreover, I'm asking myself if it is ok using the async API `function myHook(doneCallback){}` hook's interface instead of "fire&forget"

Idea?
the cons here is that any errors are ignored and will not popped in the `ready(err)`

Reply:
I start to have a feeling that the `preReady` event should really be an `onReady()` queue similar to `.after()`, but added and fired at the very end of the boot phase.